AnnoyancesFilter/sections/popups.txt

```diff
@@ -91,6 +91,8 @@ chita.ru##.chitaru__newsletter-subscription-button
 e.budgetnik.ru##body > div.backdrop
 beauty.ua##.popup-instagramm
 cosmo.com.ua##body > div.pose
+kalypsomedia.com###popdup
+kalypsomedia.com##.popup--link
```

Comment:
Unsure about this last rule.

Reply:
It is ok. That button must open this popup.